backend/courses/models.py

```diff
@@ -1,3 +1,4 @@
+from genericpath import exists
```

Comment:
Might want to move these models/views/urls under the plan folder, as they are more related to PCP specifically than courses. Courses is more for shared course-related functionality.

Reply:
My reasoning for keeping all friendship-related stuff in courses/ was because we might use this functionality in PCR (and possibly PCA too) so it would be a good idea to organize it under courses/ for use in all courses products.

Reply:
Hmm interesting, I guess that makes sense.